### ISSUE TITLE
Minion Actions

### ISSUE THREAD

**Issue by maldrasen** [maldrasen/artificer#78]
We should add something to the minion panel that lets you interact with the minion. Limit this to once per day somehow. You can probably only interact with a few minions every day as well. I'm thinking there are many events, each of which is designed to raise a specific aspect, fear, loyalty, the length of their anal prolapse, that sort of thing.

**Comment by maldrasen** [comment]
This is going to need a bit (lot) of design work before I'm really ready to go into this, but I do think I should start working on it next. I think when you start this action it needs to queue an event. The event starts with your minion summoned to you in whatever location you're in. So the location has a big influence on what actions are available. This there a bed in the room? Dirt floor, all that. I think each location somehow needs to include a list of possible actions.

Then there are also the possible actions for each minion. What are they willing / able to do? All of this needs to tie into the personality system, so all of that needs to be started as well. I might make this an epic. It's looking like a month of work or so to implement I think.

**Comment by maldrasen** [comment]
Changed my mind on this already. Instead of queuing an event, and then working through some kind of decision tree, I'm just going to show a menu of all available actions for that minion in that location. I made a SummonAction form that governs this, sets requirements and such. When a summon action is selected, then it queues the associated event. Actions can also have variants depending on the specific character and location properties and randomly chosen, but if an action is possible there should always be a default event available.
